crates/rome_js_formatter/tests/specs/ts/class/readonly_ambient_property.ts

```diff
@@ -0,0 +1,7 @@
+declare class A {
+	readonly prop = "value"
+}
+
+class A {
+	declare readonly prop = "value"
+}
```

Comment:
It would be more useful to have "not formatted" code, so we can see if the formatter actually works as intended. 

```suggestion
declare class A {


	readonly prop = "value"
	
	
}

class A {
	declare readonly prop = "value__value__value__value__value__value__value__value__value__value__value__"
}
```